src/UI/Art/SpriteArtist/SpriteLibrary.java

```diff
@@ -24,6 +24,8 @@
   // This is the physical size of a single map square in pixels.
   public static final int baseSpriteSize = 16;
 
+  static final String charKey = "%./-~,;:!?'&()";
```

Comment:
Is `package private` the intended scope here?

Reply:
I guess it could be proper private? Doesn't seem useful to other classes